src/Faithlife.WebRequests/WebServiceRequestBase.cs

```diff
@@ -188,12 +188,34 @@ protected virtual void OnWebRequestCreated(HttpRequestMessage request)
 		{
 		}
 
-		private HttpRequestMessage CreateWebRequest(out HttpClient client)
+
+		private HttpClientHandler CreateHttpClientHandler(WebServiceRequestSettings settings)
 		{
-			WebServiceRequestSettings settings = Settings ?? new WebServiceRequestSettings();
-			var request = new HttpRequestMessage(new HttpMethod(Method ?? "GET"), RequestUri);
 			var handler = new HttpClientHandler();
-			client = new HttpClient(handler);
+			if (settings.CookieManager != null)
+				handler.CookieContainer = settings.CookieManager.CookieContainer;
+
+			handler.AutomaticDecompression = DecompressionMethods.Deflate | DecompressionMethods.GZip;
+
+			if (DisableAutoRedirect)
+				handler.AllowAutoRedirect = false;
```

Comment:
👍 I confirmed that the [default is `true`](https://technet.microsoft.com/sr-latn-rs/library/system.net.http.httpclienthandler.allowautoredirect(v=vs.110)).